src/org/opendatakit/briefcase/reused/http/CommonsHttp.java

```diff
@@ -147,6 +151,16 @@ public void unsetProxy() {
       throw new HttpException("The connection has timed out", e);
     } catch (IOException e) {
       throw new UncheckedIOException(e);
+    } finally {
+      try {
+        UncheckedFiles.closeInputStream(request.getBody());
+      } catch (final BriefcaseException exception) {
+        log.error("Error: Attempted to close Request.body InputStream, but failed", exception);
+      }
```

Comment:
We don't throw `BriefcaseException` anymore in that method.

```suggestion
      UncheckedFiles.closeInputStream(request.getBody());
```

Reply:
Turns out that I was wrong and we do throw `BriefcaseException`, but it's when trying to access the body of an empty request. I've fixed that in a new commit.